packages/eslint-config-terra/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@cerner/eslint-config-terra",
+  "version": "3.10.0",
+  "description": "Terra Eslint Config",
+  "main": "eslint.config.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/eslint-config-terra"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```

Comment:
should we drop 8 support?

Reply:
Does Orion have a timeline for dropping support for the docker images running on node 8? I agree with dropping support for node 8 as soon as we can, but it might cause some rioting internally. A lot of teams need to update their nvmrc file to correspond with the version used in their CI image. (And a lot of teams are using old docker images, but this could be their push)

Reply:
Since this is a MVB it could make sense for them to do the update along with this.  Thoughts?

Reply:
+1, it's time to go to node 10